source/agora/test/EnrollmentManager.d

```diff
@@ -34,10 +34,12 @@ import core.time;
 unittest
 {
     // generate 1007 blocks, 1 short of the enrollments expiring.
```

Comment:
We should update this comment. generates 9 blocks.

Reply:
Added a commit